cloud/endagaweb/views/dashboard.py

```diff
@@ -286,6 +286,14 @@ def get(self, request, imsi=None):
                 'date')[0].date
         except IndexError:
             context['created'] = None
+        try:
+            # Get balance expire date for subscriber's first number.
+            number_details = Number.objects.filter(
```

Comment:
Fixed in commit# https://github.com/facebookincubator/CommunityCellularManager/pull/41/commits/62e328db029c5dd045566a75b012918ca278eff1